roles/sbs/defaults/main.yml

```diff
@@ -13,7 +13,7 @@ sbs_db_name: "sbs"
 sbs_db_user: "sbs"
 sbs_dbbackup_user: "sbs_backupper"
 
-sbs_redis_host: "localhost"
+sbs_redis_host: "redis.vm.scz-vm.net"
```

Comment:
Dit hoort in de vm group vars denk ik.  Het is sowieso handiger om de redis in een aparte PR te stoppen; die moet sneller worden gemerged.